### DISCUSSION TITLE
<DatePicker />  date.clone is not a function

### DISCUSSION THREAD

**Discussion by radrianhp** [ant-design/ant-design#35437] [Q&A] [answered]
I have same similar issue in https://github.com/ant-design/ant-design/issues/20206 

but I dont have solution about the issue 

Error : date.clone is not a function
When first render page using useeffect I want have value date using datepicker 

[https://codesandbox.io/s/datepicker-antd-60l5zs?file=/src/App.js](url)

**Comment by huntdream** [comment, marked as answer]
<img width="1294" alt="image" src="https://user-images.githubusercontent.com/7903755/168204293-a4f356f0-7568-45bd-b5e5-1b7061a510e9.png">
DatePicker's value only accept moment, don't convert moment object to string
